src/content/docs/ai-monitoring/explore-ai-data/view-model-data.mdx

```diff
@@ -0,0 +1,171 @@
+
+### Errors tab [#errors-inventory]
+
+<img
+    title="Model inventory: Errors"
+    alt="A screenshot of the Errors time series and chart"
+    src={aiModelInventoryErrors}
+/>
+
+<figcaption>
+    Go to **<DoNotTranslate>[one.newrelic.com](https://one.newrelic.com) > All Capabilities > AI monitoring > Model inventory > Errors</DoNotTranslate>**: View data about AI model errors.
+</figcaption>
+
+The errors tab uses time series graphs and tables to organize model errors. 
+
+* <DoNotTranslate>**Response errors**</DoNotTranslate>: Track the number of errors in aggregate that come from your AI model.
+* <DoNotTranslate>**Response errors by model**</DoNotTranslate>: Determine if one specific model produces more errors on average, or if one specific error is occurring across your models.
+* <DoNotTranslate>**Response errors by type**</DoNotTranslate>: View how often certain errors appear. 
+* <DoNotTranslate>**Errors table**</DoNotTranslate>: View error type and message in context of the request and response. 
+
+### Performance tab [#performance-inventory]
+
+<img
+    title="Model inventory: Performance"
+    alt="A screenshot of the Errors time series and chart"
+    src={aiModelInventoryPerformancePage}
+/>
+<figcaption>
+    Go to **<DoNotTranslate>[one.newrelic.com](https://one.newrelic.com) > All Capabilities > AI monitoring > Model inventory > Performance</DoNotTranslate>**: View data about your AI model's performance.
+</figcaption>
+
+The performance tab aggregates response and request metrics across all your models. Overview the models that take the most time to process a request or create a response with the pie charts, or refer to the time series graphs to track upticks in request or response times. 
+
+* The 95th percentile, or **p95**, indicates the number of transactions that were below a certain value during a time period. If you've filtered your time series to p95 and see that a call at 9:00pm took six seconds to complete, it means that 95% of all calls were below six seconds, but 5% of calls were above six seconds.
+* The 99th percentile, or **p99**, indicates that 99% of calls or transactions were below a given value during a time period. Inversely, it means 1% of transactions were above the value. 
+
+You can use performance charts to locate outliers across your models. 
+
+### Cost tab [#cost-inventory]
+
+<img
+    title="Model inventory: Performance"
+    alt="A screenshot of the Errors time series and chart"
+    src={aiModelInventoryCostPage}
+/>
+<figcaption>
+    Go to **<DoNotTranslate>[one.newrelic.com](https://one.newrelic.com) > All Capabilities > AI monitoring > Model inventory > Cost</DoNotTranslate>**: View data about your AI model's cost.
+</figcaption>
+
+The cost page uses a combination of time series graphs and pie charts to identify cost drivers amongst your models. Determine whether an increase in cost came from prompts or completions, or if certain models cost more on average than others.  
```

Comment:
Here I see `The cost page`, but  you've used `tab` for Overview, Errors, and Performance. I'd use the same one for all of them.

Reply:
Good catch! thank you